src/OpenSage.Game/Data/Ini/WindowTransition.cs

```diff
@@ -87,4 +124,62 @@ public enum WindowTransitionStyle
         [IniEnum("CONTROLBARARROW")]
         ControlBarArrow
     }
+
+    [AddedIn(SageGame.Bfme)]
+    public enum TransitionType
```

Comment:
Are you sure these are a different thing from WindowTransitionStyle?

Reply:
True, this seems to be the same stuff, 
will fix it